Description: Fix platform detection on Debian gnukfreebsd >= 9.
Author: Steven Chamberlain <steven@pyro.eu.org>
Forwarded: https://github.com/martine/ninja/pull/770

--- a/platform_helper.py
+++ b/platform_helper.py
@@ -19,7 +19,7 @@
 
 def platforms():
     return ['linux', 'darwin', 'freebsd', 'openbsd', 'solaris', 'sunos5',
-            'mingw', 'msvc', 'gnukfreebsd8']
+            'mingw', 'msvc', 'gnukfreebsd']
 
 class Platform( object ):
     def __init__( self, platform):
@@ -31,7 +31,7 @@
             self._platform = 'linux'
         elif self._platform.startswith('freebsd'):
             self._platform = 'freebsd'
-        elif self._platform.startswith('gnukfreebsd8'):
+        elif self._platform.startswith('gnukfreebsd'):
             self._platform = 'freebsd'
         elif self._platform.startswith('openbsd'):
             self._platform = 'openbsd'
